<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmergencyCoreCooling</class>
 <widget class="QDialog" name="EmergencyCoreCooling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emergency Core Cooling</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupValve">
     <property name="title">
      <string>ECC Cooling Valve</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonValveAuto">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonValveOpen">
        <property name="text">
         <string>Open</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineValve2"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineValve1"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonValveClose">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelValveFlow">
        <property name="text">
         <string>ECC Cooling Flow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="lineValveFlow"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupPump1">
     <property name="title">
      <string>Pump 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonPump1Stop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonPump1Start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPump1Flow">
        <property name="text">
         <string>Flow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="linePump1Flow"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupPump2">
     <property name="title">
      <string>Pump 2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonPump2Start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonPump2Stop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPump2Flow">
        <property name="text">
         <string>Flow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="linePump2Flow"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupDiesel">
     <property name="title">
      <string>Diesel Pump</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="2">
       <widget class="QPushButton" name="buttonDieselStart">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonDieselInc">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineDieselFlow"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelDieselFlow">
        <property name="text">
         <string>Flow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonDieselDec">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLineEdit" name="lineDieselSetpoint"/>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="labelDieselSetpoint">
        <property name="text">
         <string>Speed Setpoint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QLabel" name="labelDieselSpeed">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QLineEdit" name="lineDieselSpeed"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="buttonDieselStop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
